crossplane-contrib/provider-jet-azure
# Add Conn details SQL Server and Add mssql

### Description of your changes

This PR configures Azure SQL Server (`azurerm_sql_server`) and adds Azure MSSQL Server (`azurerm_mssql_server`) resource (that is proposed to be used instead of `azurerm_sql_server` in [the documentation](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/sql_server)) with proper connection detail keys so that they can be consumed further (e.g. in https://github.com/crossplane-contrib/provider-sql)

This PR is on top of #83, see the last commit(s) that I've authored. 

I have:


### How has this code been tested

1. Create Azure SQL Server
2. Verify that connection details secret created with expected keys and data

[contribution process]: https://git.io/fj2m9
